src/custom-elements/src/components/expansion-panel/expansion-panel.tsx

```diff
@@ -0,0 +1,89 @@
+import { Component, Event, EventEmitter, Prop } from '@stencil/core';
+
+@Component({
+    tag: 'chi-expansion-panel',
+    styleUrl: 'expansion-panel.scss',
+    scoped: true
+})
+export class ExpansionPanel {
+    /**
+     * to set expansion panel to 'done' state
+     */
+    @Prop({ reflectToAttr: true }) done?: boolean;
+
+    /**
+     * to set expansion panel to 'active' state
+     */
+    @Prop({ reflectToAttr: true }) active?: boolean;
+
+    /**
+     * to set expansion panel to 'disabled' state
+     */
+    @Prop({ reflectToAttr: true }) disabled?: boolean;
+
+    /**
+     * to set a step number next to the title of the panel
+     */
+    @Prop({ reflectToAttr: true }) step?: string;
```

Comment:
This attribute is correctly set as optional.

Reply:
@dani-cl-madrid @SergioQCL I don't really see any importance of optionals here because it's not a functional component where we kind of pass props as params where optionals might be suitable, but I highly doubt their usage in class based components. Thoughts?

Reply:
Yeah @srinath-koneru. You made me change my mind. Stencil itself manages unset values property. There is no need to put it as optional here. @SergioQCL update this.